app/views/layouts/listnav/_container_link_to.html.haml

```diff
@@ -0,0 +1,43 @@
+- return if object.nil?
```

Comment:
This logic-to-markup ratio is much more suited to a helper than a partial. Given how we're invoking it, that'll make the call sites look nicer too. How about a nicely symmetrical `listnav_link`?